src/index.html

```diff
@@ -1,13 +1,538 @@
+                        <div class="slide__wrapper">
+                            <i class="slide__icon far fa-user"></i>
+
+                            <h1 class="slide__title">Nunc digno lorem dol 2</h1>
+
+                            <p class="slide__text">Vivamus vestibulum nulla</p>
+
+                            <a class="slide__btn">laoreet ipun</a>
+                        </div>
+                    </div>
+
+                    <div class="slider__slide jsSlide">
+                        <div class="slide__wrapper">
+                            <i class="slide__icon far fa-user"></i>
+
+                            <h1 class="slide__title">Nunc digno lorem dol 3</h1>
+
+                            <p class="slide__text">Vivamus vestibulum nulla</p>
+
+                            <a class="slide__btn">laoreet ipun</a>
+                        </div>
+                    </div>
+                </div>
+                <!-- SLIDER -->
+            </div>
+        </header>
+        <!-- HEADER -->
+
+        <!-- MAIN -->
+        <main class="main">
+            <div class="main__inner">
+                <!-- INFO-TILES -->
+                <div class="info-tile-container">
+                    <div class="info-tile">
+                        <i
+                            class="info-tile__icon fa fa-headphones"
+                            aria-hidden="true"
+                        ></i>
+                        <h2 class="info-tile__title">ibus tortor</h2>
+                        <p class="info-tile__text">Morbi interdum mollis</p>
+                    </div>
+                    <div class="info-tile">
+                        <i
+                            class="info-tile__icon fa fa-sync"
+                            aria-hidden="true"
+                        ></i>
+                        <h2 class="info-tile__title">aliquam</h2>
+                        <p class="info-tile__text">Vestibulam dipbus</p>
+                    </div>
+                    <div class="info-tile">
+                        <i
+                            class="info-tile__icon fa fa-clock"
+                            aria-hidden="true"
+                        ></i>
+                        <h2 class="info-tile__title">Nunciden</h2>
+                        <p class="info-tile__text">Idot crasorma elinit</p>
+                    </div>
+                    <div class="info-tile">
+                        <i
+                            class="info-tile__icon fa fa-comments"
+                            aria-hidden="true"
+                        ></i>
+                        <h2 class="info-tile__title">egestasmod</h2>
+                        <p class="info-tile__text">Felis moodo tortor</p>
+                    </div>
+                </div>
+                <!-- INFO-TILES -->
+
+                <!-- MAIN__CONTENT -->
+                <div class="main__content">
+                    <div class="content__post-container">
```

Comment:
btw I would name that `posts` and `post__item` for every post. or `dashboard` and `dashboard__post` to avoid the usage of word `item` since it can overwhelm and you will be talking to your team about the items. But you can assign the name for the specific area in the project so everybody will understand you from the first word.